CONTRIBUTING.md

```diff
@@ -4,126 +4,79 @@
 
 # Otter Library - How to contribute
 
-Your contribution is essential to enhance the Otter repository. We want to keep it as easy as possible to provide bug fixes and whole new components that will enable you to build your web application.
+Thank you for considering contributing to Otter! We welcome all contributions, and appreciate your effort in improving our project. To ensure a smooth contribution process, please follow the guidelines below.
 
-We are always looking for contributions and will be happy to review your Pull Requests as long as those adhere to some basic rules.
+## Feature Requests
 
-## 1 - Getting started - Contribution handshake
+Please create a new [issue](https://github.com/AmadeusITGroup/otter/issues/new/choose) on our repository and choose "Feature request". This will allow us to review and prioritize your request accordingly.
 
-Before starting writing a new component, the first thing to do is to contact [our fantastic support team][contact]. Indeed, you have to make sure the contribution you are about to start has not already been provided and that your intended component or bug fix will fit inside the repository. We can then help you assess any dependencies or impacts on any other components.
-The last thing we want is to decline your pull request so please do not hesitate to contact us beforehand to agree on the entry criteria:
+## Submitting an Issue
 
-* the functional purpose/structure of the component (to avoid duplication of blocks)
-* the technical architecture (to assure the homogeneity of the code)
-* the Test Matrix for the component tests.
+Before you submit an issue, please make sure the issue is not already in our [issues backlog](https://github.com/AmadeusITGroup/otter/issues).
 
-From a QA point of view, what will interest us here is the test matrix.
-The QA members of the contributing team must provide the list of UI tests to be done at block level.
-This test matrix is reviewed and validated by Otter Core QA.
-The scope of these tests is:
+The issue creation template requires:
+- a description of the failing use case
+- the version of library where the issue occurs
+- NodeJS version
```

Comment:
this is aligned with the files in `.github/ISSUE_TEMPLATE`?

Reply:
Yes, and it complements by requesting the used NodeJS version